Improve German translation

steffenrapp opened the pull request:
Improve German translation by adding missing translations and small fixes.

Scorpoon reviewed (approved):
Looks good for me.
Handling "Sie" & "Du" is more a personal taste I guess, I'm fine with both.

Scorpoon commented on Europe/de.json:
```diff
@@ -133,44 +146,50 @@
+    "Enable_OIDC_for_Root_User": "OIDC für Root-Benutzer aktivieren",
     "Enabling_this_may_cause_performance_issues": "Die Aktivierung dieser Option kann Leistungsprobleme verursachen",
     "Expenses_and_Distance_Traveled_by_Month": "Ausgaben und zurückgelegte Entfernung nach Monat",
     "Expenses_by_Month": "Ausgaben nach Monat",
```
Isn't "Ausgaben pro Monat" more ideal?

Scorpoon commented on Europe/de.json:
```diff
@@ -133,44 +146,50 @@
+    "Enable_CSV_Imports": "CSV-Importe aktivieren",
+    "Enable_OIDC_for_Root_User": "OIDC für Root-Benutzer aktivieren",
     "Enabling_this_may_cause_performance_issues": "Die Aktivierung dieser Option kann Leistungsprobleme verursachen",
     "Expenses_and_Distance_Traveled_by_Month": "Ausgaben und zurückgelegte Entfernung nach Monat",
```
Isn't "Ausgaben und zurückgelegte Entfernung pro Monat" more ideal?

Scorpoon commented on Europe/de.json:
```diff
@@ -21,6 +22,8 @@
     "About": "Über",
     "Account_Username": "Konto-Benutzername",
+    "Adaptive_Color_Mode": "Adaptiver Farbmodus",
+    "Add/Remove_Extra_Fields": "Zusätzliche Felder hinzufügen/entfernen",
```
Can be removed, I haven't found any references to it in the code

Scorpoon commented:
And you should merge the incoming changes, "Cost" and "Cost_of_tax_paid" has been added, maybe some others too.

For "Cost" I would suggest "Gesamtpreis" instead of "Kosten".
For "Cost_of_tax_paid" I would suggest "Betrag der **b**ezahlten Gebühr" instead of "Betrag der **g**ezahlten Gebühr".

steffenrapp commented:
Thanks for your review. I hope I addressed all of your comments. With some I didn't agree but I implemented most of your suggestions.
I think "Gesamtpreis" instead of "Kosten" for "Cost" doesn't really fit.
Please have a look again.

auer1329 reviewed (commented):
The following strings were intentionally interchanged:
```
    "Max_Fuel_Economy": "Maximaler Durchschnittsverbrauch",
    "Min_Fuel_Economy": "Minimaler Durchschnittsverbrauch",
```

Due to the "miles per gallon" calculation, the min and max values are reversed in the German translation

![durchschnitt](https://github.com/user-attachments/assets/0b23dd90-7888-4d8d-acbc-8ea1a41bd0a5)

Personally I do not like "Du" instead of "Sie" but since you already made the changes I suppose we leave it as it is.


"Antriebstyp" instead of "Kraftstoffart" was also intentional, since "Kraftstoffart" does not really represent the option "Electric".

I did all my translations on my private instance of weblate on https://weblate.sciecne.org/ (you should be able to register a new account). It directly pulls the newest translation source file after every commit and allows to export the new translations with one click.

Best practice would be to collaborate on this platform to make it easier to see other suggestions and vote for the most fitting.

I uploaded your current translation to the weblate instance and will continue to work with this state.
